docs/Examples/BbhInitialData/BbhInitialData.ipynb

```diff
@@ -0,0 +1,463 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Distributed under the MIT License.\n",
+    "# See LICENSE.txt for details.\n",
+    "\n",
+    "import os\n",
+    "import h5py\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from ruamel.yaml import YAML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%bash\n",
+    "# Clean up output files from previous runs\n",
+    "rm -f Bbh*.h5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Binary black hole initial data\n",
+    "\n",
+    "In this example we run the elliptic solver to compute initial data for binary\n",
+    "black holes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, compile the `SolveXcts` executable and put in the path to your build\n",
+    "directory below. Make sure you compile in `Release` mode."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "SPECTRE_BUILD_DIR = \"/work/nfischer/spectre/build-Default-Release/\"\n",
+    "SPECTRE_HOME = os.path.abspath(\"../../../\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "## Setup input file\n",
+    "\n",
+    "We set up an input file based on the\n",
+    "`tests/InputFiles/Xcts/BinaryBlackHole.yaml` example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load example input file\n",
+    "load_input_file_path = os.path.join(\n",
+    "    SPECTRE_HOME, 'tests/InputFiles/Xcts/BinaryBlackHole.yaml')\n",
+    "yaml = YAML()\n",
+    "with open(load_input_file_path, 'r') as open_input_file:\n",
+    "    input_file = yaml.load(open_input_file)\n",
+    "# Set output file names\n",
```

Comment:
total nitpick, but I would've found this easier to parse if you have a line before this comment. Just so that the modification is clearly different from the reading. Maybe even say, `Now that we've read the input file, update the output file names, i.e. those that are dumped by the SolveXcts executable`